types/pg/index.d.mts

```diff
@@ -0,0 +1,34 @@
+import type * as pg from "./index.d.ts";
+
+import TypeOverrides = require("./lib/type-overrides");
+
+export type * from "./index.d.ts";
+
+export const defaults: typeof pg.defaults;
+export const Client: typeof pg.Client;
```

Comment:
The const exports caused a regression in my project. Before, a type could be imported using `import type {Client} from 'pg'`. After this was released my project broke because it's with the following error:

> error TS2749: 'Client' refers to a value, but is being used as a type here. Did you mean 'typeof Client'?

I've worked around it by importing the full pg namespace, but I thought I'd inform you of this regression.

Reply:
Correction, that workaround does not work. I cannot import just the types anymore. Quite an inconvenient regression.

In addition to the error described above, when importing as `import type pg from 'pg'`, you get the error: ` Cannot find namespace 'pg'.`. This is also true for not importing it as a type.